src/components/Dashboard/Dashboard.js

```diff
@@ -13,48 +13,73 @@ class Dashboard extends Component {
       list: [],
       isOpenModal: false,
       selectedCandidate: {},
-      compName : null
+      compName: null
     };
   }
 
-  handleopenModal(row,buttonName) {
-    this.setState({ isOpenModal: true, selectedCandidate: row , compName : buttonName});
+  handleopenModal(row, buttonName) {
+    this.setState({ isOpenModal: true, selectedCandidate: row, compName: buttonName });
   }
 
   handleCloseModal(row) {
-    this.setState({ isOpenModal: false, selectedCandidate: {} , compName : null});
+    this.setState({ isOpenModal: false, selectedCandidate: {}, compName: null });
   }
 
   cellButtonForL1(cell, row) {
     return (
-      <button onClick={() => this.handleopenModal(row,'L1')} id="btnopenModel">
+      <button className="btn" onClick={() => this.handleopenModal(row, 'L1')} id="btnopenModelL1">
         L1
       </button>
     );
   }
   cellButtonForGK(cell, row) {
     return (
-      <button onClick={() => this.handleopenModal(row,'GK')} id="btnopenModel">
+      <button className="btn" onClick={() => this.handleopenModal(row, 'GK')} id="btnopenModelGK">
         GK
       </button>
     );
   }
+  dataToDashboard = (recivedObjData) => {
+    this.setState({
+      formdata: [...this.state.list, recivedObjData]
```

Comment:
I dont think we need form data in state

Reply:
this data is no longer in use so removing out those lines of code